desktop/apps/artist/test/components/current_venice_feature/index.coffee

```diff
@@ -0,0 +1,32 @@
+currentItem = require "../../../components/current_venice_feature"
+{ isNull } = require "underscore"
+Q = require "bluebird-q"
+
+describe "current venice feature", ->
+  fetch = () -> Q.resolved [
+    { slug: "featured", featured: true },
+    { slug: "participant", featured: false }
+  ]
+
+  it "renders nothing if the given artist is not in the list of artists", ->
+    currentItem({ id: "not-featured" }, fetch).then (result) ->
+      (result == undefined).should.equal true
+
+  it "renders an artist featured in editorial content", ->
+    currentItem({ id: "featured", name: "Banksy" }, fetch).then (result) ->
+      result.heading.should.match(/INSIDE/)
```

Comment:
also cool! I'd used `containEql` for a substring comparison, looks like this is 👍